components/script/dom/offscreencanvas.rs

```diff
@@ -214,6 +215,36 @@ impl OffscreenCanvasMethods<crate::DomTypeHolder> for OffscreenCanvas {
         }
     }
 
+    /// <https://html.spec.whatwg.org/multipage/#dom-offscreencanvas-transfertoimagebitmap>
+    fn TransferToImageBitmap(&self, can_gc: CanGc) -> Fallible<DomRoot<ImageBitmap>> {
+        // Step 2. If this OffscreenCanvas object's context mode is set to none,
```

Comment:
I would still put Step 1 here even to write a short explanation of why it isn't necessary or to add a TODO.

Reply:
There are another PR (in parallel) which will add `Step 1` (https://github.com/servo/servo/pull/37872) - depends from will it be reviewed and merged into main branch before of after this PR.
Any way I added `TODO Step 1` just in case.